server/mailet/dkim/src/main/java/org/apache/james/jdkim/mailets/DKIMSign.java

```diff
@@ -159,14 +159,11 @@ private PrivateKey extractPrivateKey(InputStream rawKey, char[] passphrase) thro
         Object pemObject = pemParser.readObject();
         JcaPEMKeyConverter converter = new JcaPEMKeyConverter().setProvider("BC");
         KeyPair keyPair;
-        if (pemObject instanceof PEMEncryptedKeyPair)
-        {
+        if (pemObject instanceof PEMEncryptedKeyPair) {
             PEMEncryptedKeyPair pemEncryptedKeyPair = (PEMEncryptedKeyPair) pemObject;
             PEMDecryptorProvider decProv = new JcePEMDecryptorProviderBuilder().build(passphrase);
             keyPair = converter.getKeyPair(pemEncryptedKeyPair.decryptKeyPair(decProv));
-        }
-        else
-        {
+        } else {
```

Comment:
are we sure it will be a PEMKeyPair in this case ?

Reply:
PEM can get anything :/